src/Arcus.BackgroundJobs.Tests.Integration/Jobs/DatabricksJobMetricsJobTests.cs

```diff
@@ -73,15 +74,8 @@ public async Task FinishedDatabricksJobRun_GetsNoticedByRepeatedlyDatabricksJob_
 
             using (var client = DatabricksClient.CreateClient(baseUrl, token))
             {
-                var parameters = new[]
-                {
-                    new KeyValuePair<string, string>("RequestId", "1"),
-                    new KeyValuePair<string, string>("SenderName", "ArcusSender"),
-                    new KeyValuePair<string, string>("ThrowError", "False")
-                };
-
                 // Act
-                await client.Jobs.RunNow(2, RunParameters.CreateNotebookParams(parameters));
+                await client.Jobs.RunNow(9, RunParameters.CreateNotebookParams(Enumerable.Empty<KeyValuePair<string, string>>()));
```

Comment:
Maybe we can move the job id to the configuration?

Reply:
Yes, was also an idea; I just wanted here to check if my Databricks job I created works here. But yes, will do.

Reply:
Voila, moved to YAML variables.